dbt/include/databricks/macros/materializations/incremental/incremental.sql

```diff
@@ -38,6 +38,10 @@
     {%- endcall -%}
     {% do persist_constraints(target_relation, model) %}
     {% do apply_tags(target_relation, tags) %}
+    {%- if language == 'python' -%}
```

Comment:
If were a creating a new table (rather than incremental), we need to apply tblproperties to python models, since for now there is no python API to use.